spec/features/download_panel_spec.rb

```diff
@@ -32,7 +32,23 @@
       page.find('[data-sul-embed-toggle="sul-embed-download-panel"]', match: :first).click
       within '.sul-embed-download-list' do
         expect(page).to have_css '.sul-embed-download-list-item a.download-link', text: 'Download Thumbnail'
-        page.save_screenshot('tmp/screen.png')
```

Comment:
And yep this would get saved at `tmp/screen.png` https://github.com/teampoltergeist/poltergeist#whats-supported